deploy/docs/Upgrading_with_Helm.md

```diff
@@ -0,0 +1,78 @@
+
+
+```bash
+
+helm upgrade [RELEASE] [CHART] [flags]
+
+```  
+
+Some useful Options
+
+|     Option     	| Description                                                                                                                                              	|
+|:--------------:	|----------------------------------------------------------------------------------------------------------------------------------------------------------	|
+|     `--debug`    	| Enable verbose output                                                                                                                                    	|
+|    `--dry-run`   	| Simulate an upgrade                                                                                                                                      	|
+|   `-h`, `--help`   	| help for upgrade                                                                                                                                         	|
+|   `--no-hooks`   	| Disable pre/post upgrade hooks                                                                                                                           	|
+| `--reset-values` 	| When upgrading, reset the values to the ones built into the chart                                                                                        	|
+| `--reuse-values` 	| When upgrading, reuse the last releases values and merge in any overrides from the command line via `--set` and `-f`                                        	|
+|  `--set` stringArray	| Set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)                                          	|
+|   `--set-file` stringArray  	| Set values from respective files specified via the command line (can specify multiple or separate values with commas: key1=path1,key2=path2) 	|
+|  `--set-string` stringArray  	|  Set STRING values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)                             	|
+|    `--version`   	| Specify the exact chart version to use. If this is not specified, the latest version is used                                                             	|
+
+
+To edit or append to the existing customized values, add the `–reuse-values` flag, otherwise any existing customized values are ignored.
+
+If no chart value arguments are provided on the command line, any existing customized values are carried forward, i.e. the `--reuse-values` flag will be used by default. If you want to revert to just the values provided in the chart, use the `-–reset-values` flag.
+
+And if there exists `--set/--set-file/--set-string/--values/-f` , the `--reset-values` flag will be used by default.
+
+### Example using the `--set` flag in the upgrade command:
+
+```bash
+helm upgrade collection sumologic/sumologic --set key1=val1, key2=val2
```

Comment:
shouldn't we advise setting `--reuse-values` here? It will fail for users unless they happen to be setting access key/id/endpoint.

Reply:
+1, let's add `--reuse-values` in the example